transactions/src/main/resources/schema/wildfly-txn_3_0.xsd

```diff
@@ -170,7 +170,7 @@
             </xs:documentation>
         </xs:annotation>
         <xs:attribute name="path" type="xs:string" default="tx-object-store"/>
-        <xs:attribute name="relative-to" type="xs:string" default="jboss.server.data.dir" />
+        <xs:attribute name="relative-to" type="xs:string"/>
```

Comment:
If a new schema is being added can we please add the new schema in one commit and then make the new changes in a second commit?